<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curve3DDock</class>
 <widget class="QWidget" name="Curve3DDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="labelX">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="TreeViewComboBox" name="cbYCoordinate"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPointHeader">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Point</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelY">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="cbClosedCurve">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Closed curve</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="KLineEdit" name="leName"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <widget class="TreeViewComboBox" name="cbXCoordinate"/>
       </item>
       <item row="6" column="3">
        <widget class="TreeViewComboBox" name="cbZCoordinate"/>
       </item>
       <item row="1" column="3">
        <widget class="KLineEdit" name="leComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="cbShowEdges">
         <property name="text">
          <string>Show edges</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelZ">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Point size</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QDoubleSpinBox" name="sbPointSize">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>commonfrontend/widgets/TreeViewComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leComment</tabstop>
  <tabstop>cbXCoordinate</tabstop>
  <tabstop>cbYCoordinate</tabstop>
  <tabstop>cbZCoordinate</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
